reposerver/src/test/scala/com/advancedtelematic/tuf/reposerver/http/S3StorageResourceIntegrationSpec.scala

```diff
@@ -50,4 +51,33 @@ class S3StorageResourceIntegrationSpec extends TufReposerverSpec
       header("Location").get.value() should include("amazonaws.com")
     }
   }
+
+  test("uploading a target changes targets json with custom endpoint url") {
+    lazy val credentials = new S3Credentials("", "", "", Regions.fromName("us-central-1"), "https://storage.googleapis.com")
```

Comment:
This test runs with my credentials, which are just aws credentials, not google, something wrong here?